@here/harp-mapview/lib/Tile.ts

```diff
@@ -931,9 +927,9 @@ export class Tile implements CachedResource {
         this.clear();
         this.userTextElements.length = 0;
         this.m_disposed = true;
-
-        // Ensure that tile is removable from tile cache.
         this.isVisible = false;
+        // Ensure that tile is removable from tile cache.
+        this.frameNumLastRequested = 0;
```

Comment:
I thought dispose is called from the TileCache when the tile is evicted, so it is a bit weird that we have to make sure that the Tile can be evicted.

Reply:
Yes, but it is not the only place it is called from, it is called on tiles that have exited the view, search for disposeTile, so in this case, it can also be removed from the tile cache.